scripts/variantstore/AOU_GVS_WORKSPACE.md

```diff
@@ -58,20 +61,39 @@ Next, your re-blocked gVCF files should be imported into GVS by running the `Gvs
 
 These are the required parameters which must be supplied to the workflow:
 
-| Parameter      | Description |
-| ----------------- | ----------- |
-| project_id | The name of the google project containing the dataset |
-| dataset_name      | The name of the dataset you created above       |
+| Parameter             | Description |
+|-----------------------| ----------- |
+| project_id            | The name of the google project containing the dataset |
+| dataset_name          | The name of the dataset you created above       |
 | external\_sample_name | The name of the sample from the data model |
-| gvs\_sample_id | The gvs_id from the data model |
-| interval_list | The interval list to use |
+| gvs\_sample_id        | The gvs_id from the data model |
+| interval_list         | The interval list to use |
+
+**Note:**
+Samples that will be batched and loaded together must be put into a sample_set ahead of time, otherwise their loading may cause conflicts.
+This workflow must be done piecemeal if over 4000 samples are to be loaded as only 4000 samples can be loaded in at a time. The best way to do this currently is to create sample_sets of 4000 samples each.
+The workflow can then be run once for each sample_set. if the same sample_set is inadvertantly run twice the workflow will detect that the samples already exist in the system and the second duplicate workflow will fail.
+If any of the imports have failed on a single sample, check that all of the other samples have been loaded in that sample_set during that workflow. Sometimes a sample will fail during loading while there are still samples in the queue waiting for loading to begin. Because of the failure, these samples will not be loaded at all.
+Keep track of the samples that have not been loaded whether because they failed, or because they were in the queue when another sample failed. They will need to be added later.
+
+Once all sample_sets have been run, if there have been any failures, collect all non-loaded samples together in a new sample_set and load that in.
+
```

Comment:
should I make a ticket for this?

Reply:
Like the other queries that you include in this doc, you could include a sample query that lists all the samples that are in the `sample_info` table but not in the `sample_load_status`, which means the loading was never kicked off.  The user could use this instead of having to keep track during ingest in order to figure out which samples those were.